docs/openapi.yaml

```diff
@@ -580,7 +623,11 @@ paths:
         name: publicationIdentifier
         schema:
           type: string
+          format: uuid
```

Comment:
I know we do some "magic" with our identifiers to make them sortable, so it is not a standard UUID.